reports/_input.qmd

```diff
@@ -20,21 +21,21 @@ hie_pins <- training_data %>%
   pull(meta_pin) %>%
   unique()
 
-hie_data_fil <- hie_data %>%
-  filter(
-    pin %in% hie_pins,
-    year >= as.numeric(params$year) - 4,
-    qu_sqft_bld > 0
-  )
-
-chars_data_fil <- chars_data %>%
+chars_data_table <- chars_data %>%
```

Comment:
A lot of these changes are about making the report easier to work with interactively. Making the final data that feeds a given table an object that can be inspected is easier to work with than only being able to inspect the final output.